app/views/centers/show.html.slim

```diff
@@ -0,0 +1,20 @@
+h2
+  | Details of #{@center.name}
+table
+  thead
+    tr
+      th Manager Name
+      th Email
+      th Contact Number
+      th Address
+  tbody
+    tr
+      td= @center.manager_name
+      td= @center.email
+      td= @center.contact_number
+      td= @center.address
+button onclick="goBack()"  Go Back
```

Comment:
please change this to `= link_to 'Go Back', centers_path` and remove the javascripts too